test/shlibloadtest.c

```diff
@@ -117,6 +117,7 @@ static int test_lib(void)
         if (!TEST_true(shlib_load(path_crypto, &cryptolib))
                 || !TEST_true(shlib_load(path_ssl, &ssllib)))
             goto end;
+        break;
     case SSL_FIRST:
         if (!TEST_true(shlib_load(path_ssl, &ssllib))
                 || !TEST_true(shlib_load(path_crypto, &cryptolib)))
```

Comment:
It's been question on the omc list a while ago, and I can't recall that we came to a conclusion, just that some of us like it and others don't.  Ah well, we've let it go so far.

Now, to get that style into our source formatting script ;-)